source/Calamari/Commands/ReleasePackageLockCommand.cs

```diff
@@ -31,10 +26,27 @@ public ReleasePackageLockCommand(IVariables variables, IManagePackageUse package
             this.variables = variables;
             this.log = log;
             this.packageJournal = packageJournal;
+            timeBeforePackageExpiration = GetTimeBeforeExpiration(variables.Get(KnownVariables.Calamari.PackageRetentionLockExpiration));
             Options.Add("packageId=", "Package ID to download", v => packageId = v);
             Options.Add("packageVersion=", "Package version to download", v => packageVersion = v);
         }
 
+        TimeSpan GetTimeBeforeExpiration(string variable)
+        {
+            var defaultTime = TimeSpan.FromDays(14);
```

Comment:
I think this would be better as a class-level variable, renamed appropriately.

Reply:
I've moved it to a class variable called `defaultTimeBeforeLockExpiration`